src/components/Login/Login.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import './Login.scss';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faFacebook, faGoogle } from '@fortawesome/free-brands-svg-icons';
+import { useState } from 'react';
+
+const Login = ({ data, closeModal }) => {
+  const { title, subtitle, facebook, google, btn, text, footer } = data;
+  const [loginInfo, setLoginInfo] = useState({
+    email: '',
+    password: '',
+  });
+  const [idCheck, setIdCheck] = useState('');
+  const [pwCheck, setPwCheck] = useState('');
+
+  const saveLoginInfo = e => {
+    const { name, value } = e.target;
+    setLoginInfo({ ...loginInfo, [name]: value });
+  };
+
+  const idCondition =
+    loginInfo.email.includes('@') &&
+    loginInfo.email.length > 5 &&
+    loginInfo.email.includes('.');
+
+  const IdChecked = () => {
+    if (loginInfo.email.length === 0) {
+      setIdCheck('This field is required');
+    } else if (!idCondition) {
+      setIdCheck('Not a valid email address');
+    } else {
+      setIdCheck('');
+    }
+  };
+  const pwChecked = () => {
+    if (idCondition) {
+      if (loginInfo.password.length === 0) {
+        setPwCheck('This field is required');
+      } else {
+        setPwCheck('');
+      }
+    }
+  };
+
+  return (
+    <>
+      <div className="login_modal">
```

Comment:
* 최상위 태그의 선택자는 컴포넌트 이름이랑 동일하게 적는게 위코드 컨벤션 입니다.
* sass로 네스팅 할 때 모든 className을 하나로 묶고, 최대한 중복을 방지위한 장치 입니다.
```suggestion
      <div className="login">
```